README.md

```diff
@@ -13,6 +13,19 @@ ngrok is an API gateway cloud service that forwards traffic from internet-access
 - [ngrok docs][ngrok-docs]
 - [ngrok Docker Hub repository][ngrok-dockerhub]
 
+## Tags
+
+We offer the following image tags. All tags below are multi-architecture and come in 32- and 64-bit flavors for both ARM and x86.
+
+- `:latest`, :`debian` - The latest Debian-based build of the ngrok agent.
+- `:alpine` - The latest Alpine-based build of the ngrok agent.
+- `:3, :3-debian` - The latest Debian build for version 3 of the ngrok agent. Available for every major version from v3 and up.
+- `:3-alpine` - The latest Alpine build for version 3 of the ngrok agent. Available for every major version from v3 and up.
+- `:3.25.1-debian` - The latest Debian build for version 3.25.1 of the ngrok agent. Available for every minor version from 2.3.40 and up.
+- `:3.25.1-alpine` - The latest Alpine build for version 3.25.1 of the ngrok agent. Available for every minor version from 2.3.40 and up.
+- `:3.25.1-debian-d2827ad` - An immutable tag pointing to a Debian build of agent version 3.25.1 with build hash d2827ad. Available for every build we release.
+- `:3.25.1-alpine-d2827ad` - An immutable tag pointing to an Alpine build of agent version 3.25.1 with build hash d2827ad. Available for every build we release.
```

Comment:
nit: don't use a hard-coded version here, otherwise someone has to manually keep this updated every time we do a release.


```suggestion
- `:<version>-debian` - Debian builds for specific versions of the ngrok agent. Available for every minor version from 2.3.40 and up.
- `:<version>-alpine` - Alpine builds for specific versions of the ngrok agent. Available for every minor version from 2.3.40 and up.
- `:<version>-debian-d2827ad` - Immutable tags pointing to the Debian build of the ngrok agent for a specific version. Available for every build we release.
- `:<version>-alpine-d2827ad` - Immutable tags pointing to the Alpine build of the ngrok agent for a specific version. Available for every build we release.
```

Reply:
Resolved 👍